packages/remark-parse/lib/util/remove-indentation.js

```diff
@@ -56,18 +54,7 @@ function indentation(value, maximum) {
         index--
       }
 
-      if (
-        trim(values[position]).length !== 0 &&
-        minIndent &&
-        index !== minIndent
-      ) {
-        padding = tab
-      } else {
-        padding = ''
-      }
-
-      values[position] =
-        padding + values[position].slice(index in stops ? stops[index] + 1 : 0)
+      values[position] = values[position].slice(stops[index] + 1)
```

Comment:
This is not needed anymore because `stops` will now contain all indentation points, therefore `index` will never be different than `minIndent`.